### PR TITLE
Improve/simplify version variable

### PR DESCRIPTION
**Pull request by dzedro**
- Verification run:
[12-SP3](http://10.100.12.155/tests/6253)
[15](http://10.100.12.155/tests/6254)

### CODE REVIEW

File: data/qam/handle_bind_source_dependencies.sh [hunk 1]
```diff
@@ -1,13 +1,12 @@
...
 set -eox pipefail
-SOURCE_FILE=/etc/bash.bashrc.local
+. /etc/os-release
 SUSECONNECT=/tmp/SUSEConnect.txt
-VERSION=$(SUSEConnect --status-text|grep SLE[SD]/|awk -F/ '{print$2}')
-MAJOR_VERSION=$(echo $VERSION|cut -c1-2)
+MAJOR_VERSION=$(echo $VERSION_ID|cut -c1-2)
 
-if echo $VERSION|grep 15 > /dev/null 2>&1; then
+if echo $VERSION_ID|grep 15 > /dev/null 2>&1; then
```
Comment (mimi1vx) [inline data/qam/handle_bind_source_dependencies.sh]: what about function from this construction ? DRY

File: data/qam/handle_bind_source_dependencies.sh [hunk 2]
```diff
@@ -25,10 +28,10 @@ if [ -e /tmp/REMOVE_ADDED_PRODUCTS ]; then
         rm -f /tmp/REMOVE_PC
     fi
     if [ -e /tmp/REMOVE_SDK ]; then
-        SUSEConnect -d -p $SDK/$VERSION/x86_64
+        SUSEConnect -d -p $SDK/$VERSION_ID/x86_64
         rm -f /tmp/REMOVE_SDK
     fi
-    if echo $VERSION|grep 12 > /dev/null 2>&1; then
+    if echo is_sle12; then
```
Comment (kravciak) [inline data/qam/handle_bind_source_dependencies.sh]: extra echo

**Comment by dzedro** [comment]
Didn't want to use extended brackets for pattern matching

File: data/qam/handle_bind_source_dependencies.sh [hunk 3]
```diff
@@ -1,21 +1,24 @@
...
+function is_sle12 {
+    echo $VERSION_ID|grep -q 12
+}
 
 function finish {
  rm -f /tmp/SUSEConnect.txt
 }
 trap finish EXIT
 
+if echo $VERSION_ID|grep -q 15; then
```
Comment (mimi1vx) [inline data/qam/handle_bind_source_dependencies.sh]: https://www.tldp.org/LDP/abs/html/testconstructs.html#DBLBRACKETS

Comment (dzedro) [reply data/qam/handle_bind_source_dependencies.sh]: yes extended brackets are much better

**Pull request merged**